app/scripts/metamask-controller.actions.test.js

```diff
@@ -138,21 +133,21 @@ describe('MetaMaskController', function () {
         metamaskController.addNewAccount(1),
         metamaskController.addNewAccount(1),
       ]);
-      assert.equal(addNewAccountResult1, addNewAccountResult2);
+      expect(addNewAccountResult1).toStrictEqual(addNewAccountResult2);
     });
 
     it('two successive calls with same accountCount give same result', async function () {
       await metamaskController.createNewVaultAndKeychain('test@123');
       const addNewAccountResult1 = await metamaskController.addNewAccount(1);
       const addNewAccountResult2 = await metamaskController.addNewAccount(1);
-      assert.equal(addNewAccountResult1, addNewAccountResult2);
+      expect(addNewAccountResult1).toStrictEqual(addNewAccountResult2);
     });
 
     it('two successive calls with different accountCount give different results', async function () {
       await metamaskController.createNewVaultAndKeychain('test@123');
       const addNewAccountResult1 = await metamaskController.addNewAccount(1);
       const addNewAccountResult2 = await metamaskController.addNewAccount(2);
-      assert.notEqual(addNewAccountResult1, addNewAccountResult2);
+      expect(addNewAccountResult1).not.toStrictEqual(addNewAccountResult2);
```

Comment:
Can we try 
```suggestion
      expect(addNewAccountResult1).not.toEqual(addNewAccountResult2);
```
Should be sufficient

Reply:
The test fails anyway :(

```
  ● MetaMaskController › #addNewAccount › two successive calls with different accountCount give different results

    pointHex must be hex string or Uint8Array

      3914 |     const releaseLock = await this.createVaultMutex.acquire();
      3915 |     try {
    > 3916 |       return await this.keyringController.createNewVaultAndKeychain(password);
           |              ^
      3917 |     } finally {
      3918 |       releaseLock();
      3919 |     }

      at ensureBytes (node_modules/ethereum-cryptography/node_modules/@noble/curves/src/abstract/utils.ts:126:11)
      at Function.fromHex (node_modules/ethereum-cryptography/node_modules/@noble/curves/src/abstract/weierstrass.ts:318:55)
      at pubToAddress (node_modules/@ethereumjs/util/src/account.ts:285:52)
      at HdKeyring._addressfromPublicKey (node_modules/@metamask/eth-hd-keyring/index.js:306:7)
      at node_modules/@metamask/eth-hd-keyring/index.js:137:19
          at Array.map (<anonymous>)
      at HdKeyring.addAccounts (node_modules/@metamask/eth-hd-keyring/index.js:136:35)
      at KeyringController.addNewKeyring (node_modules/@metamask/keyring-controller/src/KeyringController.ts:738:21)
      at KeyringController.createKeyringWithFirstAccount_fn (node_modules/@metamask/keyring-controller/src/KeyringController.ts:1853:12)
      at KeyringController.createNewVaultWithKeyring_fn (node_modules/@metamask/keyring-controller/src/KeyringController.ts:1741:5)
      at KeyringController.createNewVaultAndKeychain (node_modules/@metamask/keyring-controller/src/KeyringController.ts:702:9)
      at MetamaskController.createNewVaultAndKeychain (app/scripts/metamask-controller.js:3916:14)
      at Object.<anonymous> (app/scripts/metamask-controller.actions.test.js:145:7)
```

Reply:
Fixed by enabling node environment using:

```typescript
/**
 * @jest-environment node
 */
```